### ISSUE TITLE
[Fixed in GH-6052] Safari: Missing Visual Focus on My Account: Consent Management

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4336]
![Screen Shot 2019-08-29 at 9 04 47 PM](https://user-images.githubusercontent.com/15113219/63986183-d7772a80-caa0-11e9-8b93-b217b90eaf9e.png)

**Comment by kacperknapik** [comment]
- Chrome: ✔
- Firefox: ✔ (note: checkbox focus border is huge)
- Safari: ❓

**Comment by KateChuen** [comment]
Chrome & Firefox: OK

Missing keyboard accessibility & visual focus on the checkboxes on Safari.

**Comment by KateChuen** [comment]
Tested on dev3. Missing visual focus on safari for:

- the checkboxes

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 1 26 16 PM](https://user-images.githubusercontent.com/15113219/73564313-46c8c000-442d-11ea-9bc1-d9826d32baa6.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome 
- Firefox
- Safari

Only issue is I could actually select CLEAR ALL and SELECT ALL #6530
